src/pages/basic_data/institution_admin/AddAffiliationSection.tsx

```diff
@@ -28,7 +28,7 @@ export const AddAffiliationSection = () => {
 
   return (
     <section>
-      <Typography variant="h3" gutterBottom>
+      <Typography component="h2" variant="h3" gutterBottom>
```

Comment:
Bevisst at vi vil at h2 skal se ut som h3? Siden dette er lagt inn flere plasser her og i den andre PRen tyder det kanskje på at vi skulle endra på stylingen til headings (om det er dette vi vil da). That said, så er det jo egentlig helt greit at de styles ulikt da, men best-practice er vel at en h2 ser ut som en h2 etc

```suggestion
      <Typography variant="h2" gutterBottom>
```